### ISSUE TITLE
loops does not work as expected with io_limit

### ISSUE THREAD

**Issue by mdodge** [axboe/fio#261]
io_limit  + loops doesn't work the way I think it should.  

I'm using it with verify, so I'd like the io_limit to be written, then verified n loop times.   Instead, the verify happens only once.   n loops * io_limit amount of data is written.  Then n loops * io_limit amount of data is verified.   This effectively changes to job itself.

**Comment by axboe** [comment]
I would just split the write part and the verify part into separate sections. Add loops=5 to the verify section. If you paste your job file here, I can help you with that.

**Comment by axboe** [comment]
Actually, does seem to behave a bit odd, with size it does what it should. Let me look into this, io_limit should be the same.

**Comment by mdodge** [comment]
Thanks Jens.  BTW, amazing tool

Well, I'm sorta using this as a more controlled/intuitive verify_backlog.  I'm testing a file system with a write log of a certain size and I want to make sure that i'm not hitting the write log on the subsequent read/verify.  I want that data to have been flushed before the verify.    I know this can be accomplished with verify_backlog.  I really want that write/verify cycle to continue.   Your suggestion would give me  write once verify 5 times.

Basically, I wish there were a mode where runtime and loops didn't interfere with the write/verify cycle when io_limit is used.
